desktop/ui/src/main/java/org/datacleaner/panels/tokenizer/TokenizerJobBuilderPresenter.java

```diff
@@ -59,21 +59,26 @@ public TokenizerJobBuilderPresenter(TransformerComponentBuilder<?> transformerJo
 	@Override
 	protected PropertyWidget<?> createPropertyWidget(ComponentBuilder componentBuilder,
 			ConfiguredPropertyDescriptor propertyDescriptor) {
-		PropertyWidget<?> propertyWidget = super.createPropertyWidget(componentBuilder, propertyDescriptor);
-		String propertyName = propertyDescriptor.getName();
+		final PropertyWidget<?> propertyWidget = super.createPropertyWidget(componentBuilder, propertyDescriptor);
+		final String propertyName = propertyDescriptor.getName();
 		if ("Token target".equals(propertyName)) {
 			_tokenTargetPropertyWidget = (SingleEnumPropertyWidget) propertyWidget;
 			_tokenTargetPropertyWidget.addComboListener(new Listener<Enum<?>>() {
-				@Override
-				public void onItemSelected(Enum<?> item) {
-					if (item == TokenTarget.ROWS) {
-						if (_numTokensPropertyWidget == null) {
-							logger.warn("No property widget for 'num tokens' found!");
-						} else if (!_numTokensPropertyWidget.isSet()) {
-							_numTokensPropertyWidget.onValueTouched(10000);
-						}
-					}
-				}
+                @Override
+                public void onItemSelected(Enum<?> item) {
+                    if (_numTokensPropertyWidget == null) {
+                        logger.warn("No property widget for 'num tokens' found!");
+                        return;
+                    }
+                    if (item == TokenTarget.ROWS) {
+                        if (!_numTokensPropertyWidget.isSet()) {
+                            _numTokensPropertyWidget.onValueTouched(2);
+                        }
+                        _numTokensPropertyWidget.setEnabled(false);
+                    } else {
+                        _numTokensPropertyWidget.setEnabled(true);
+                    }
+                }
 			});
```

Comment:
Something looks a bit screwy here. I guess it's the tabs->space change. In this case the change is IMHO a large enough part to warrant a whole-file reformatting (otherwise I'd probably reformat the method, just to make sure that braces match)